addons/binding/pom.xml

```diff
@@ -36,6 +36,7 @@
     <module>org.openhab.binding.squeezebox</module>
     <module>org.openhab.binding.tesla</module>
     <module>org.openhab.binding.vitotronic</module>
+	<module>org.openhab.binding.yamahareceiver2</module>
```

Comment:
Is there any maven command for "lint" checking the xml files? Eclipse handles the java source formatting very well but sometimes formatting issues with xml files slip through.